src/ipa_pc/data_structures.rs

```diff
@@ -235,6 +235,41 @@ impl<G: AffineCurve> ToBytes for Proof<G> {
     }
 }
 
+/// `BatchProof` generated by batching scheme.
+/// Refers to the "Multi-poly multi-point assertions" section of the "Recursive SNARKs based on Marlin" document
+#[derive(Derivative)]
+#[derivative(
+    Default(bound = ""),
+    Hash(bound = ""),
+    Clone(bound = ""),
+    Debug(bound = "")
+)]
+pub struct BatchProof<G: AffineCurve> {
+
+    /// Muli-poly single-point proof
```

Comment:
The comment is a bit misleading. I would just call it proof, and explain that it is a "classical" single-point multi-poly proof which involves all commitments (i.e. the one from the inital claim and the new "batch_commitment")

Reply:
I've changed the description of this part